cmd/kindnetd/main.go

```diff
@@ -71,7 +71,7 @@ func main() {
 	// enforce ip masquerade rules
 	ipv6 := false
 	// TODO: dual stack...?
-	if strings.Contains(hostIP, ":") {
+	if IsIPv6(hostIP) {
```

Comment:
and if we replace the if block by `ipv6 := IsIPv6(hostIP)`?

Reply:
done